src/lib/plugin.js

```diff
@@ -339,9 +339,10 @@ class FiveBellsLedger extends EventEmitter2 {
     // TODO check the timestamp the ledger sends back
     // See https://github.com/interledger/five-bells-ledger/issues/149
     if (fulfillmentRes.statusCode === 200 || fulfillmentRes.statusCode === 201) {
-      return 'executed'
+      return Promise.resolve(null)
     } else {
       this.log.error('Failed to submit fulfillment for transfer: ' + transferID + ' Error: ' + (fulfillmentRes.body ? JSON.stringify(fulfillmentRes.body) : fulfillmentRes.error))
+      throw new Error('Failed to submit fulfillment for transfer: ' + transferID + ' Error: ' + (fulfillmentRes.body ? JSON.stringify(fulfillmentRes.body) : fulfillmentRes.error))
```

Comment:
If the user is expecting a Promise to be returned, this should be `return Promise.reject(new Error('...'))`